data-processing-lib/python/requirements.txt

```diff
@@ -4,3 +4,4 @@
   argparse
   mmh3
   psutil
+  polars
```

Comment:
If this is the first module to introduce polars, consider adding a >= version constraints

Reply:
set to >=1.16.0